lib/mayaUsd/ufe/UsdContextOps.cpp

```diff
@@ -380,69 +380,53 @@ _computeLoadAndUnloadItems(const UsdPrim& prim)
         // - The current prim does not have a payload, so there must be paths
         //   below it that are still unloaded.
         itemLabelPairs.emplace_back(
-            std::make_pair(
-                kUSDLoadWithDescendantsItem,
-                kUSDLoadWithDescendantsLabel));
+            std::make_pair(kUSDLoadWithDescendantsItem, kUSDLoadWithDescendantsLabel));
     }
 
     // If anything is loaded at this prim path or any of its descendants, add
     // an item for unload.
     if (!loadedSet.empty()) {
-        itemLabelPairs.emplace_back(
-            std::make_pair(kUSDUnloadItem, kUSDUnloadLabel));
+        itemLabelPairs.emplace_back(std::make_pair(kUSDUnloadItem, kUSDUnloadLabel));
     }
 
     return itemLabelPairs;
 }
 
-}
+} // namespace
 
 namespace MAYAUSD_NS_DEF {
 namespace ufe {
 
 UsdContextOps::UsdContextOps(const UsdSceneItem::Ptr& item)
-	: Ufe::ContextOps()
+    : Ufe::ContextOps()
     , fItem(item)
 {
 }
 
-UsdContextOps::~UsdContextOps()
-{
-}
+UsdContextOps::~UsdContextOps() { }
 
 /*static*/
 UsdContextOps::Ptr UsdContextOps::create(const UsdSceneItem::Ptr& item)
 {
-	return std::make_shared<UsdContextOps>(item);
+    return std::make_shared<UsdContextOps>(item);
 }
 
-void UsdContextOps::setItem(const UsdSceneItem::Ptr& item)
-{
-	fItem = item;
-}
```

Comment:
Why this? I think its okay when this in a header file, but not in a cpp.

Reply:
I don't know of a way to have separate rules for header files vs cpp. Given it shouldn't affect our ability to debug the method, let's not re-open the discussion and keep the existing configuration.